src/TestProblem/Hydro/CCSN/Init_TestProb_Hydro_CCSN.cpp

```diff
@@ -303,6 +314,11 @@ void SetParameter()
       Aux_Message( stdout, "  central density threshold for CCSN_CC_MaxRefine_LV2 = %13.7e\n", CCSN_CC_MaxRefine_Dens2  ); }
       Aux_Message( stdout, "  central density factor for reducing dt              = %13.7e\n", CCSN_CC_CentralDensFac   );
       Aux_Message( stdout, "  reduced dt near bounce                              = %13.7e\n", CCSN_CC_Red_DT           );   }
+      if ( CCSN_CC_Form_Rot ) {
+      Aux_Message( stdout, "  rotational parameter R_0                            = %13.7e\n", CCSN_CC_Form_Rot_R_0     );
+      Aux_Message( stdout, "  rotational parameter Omega_0                        = %13.7e\n", CCSN_CC_Form_Rot_Omega_0 ); }
+      if ( CCSN_CC_Rot_Mag )
+      Aux_Message( stdout, "  magnification factor for rotational profile         = %13.7e\n", CCSN_CC_Rot_Mag_Fact     );
```

Comment:
In the case of using `CCSN_CC_Rot_Mag`, we do not use `CCSN_CC_Form_Rot` since we can use `CCSN_CC_Form_Rot_Omega_0` for the magnitude of the rotation.

Reply:
- I meant to output the value of `CCSN_CC_Rot_Mag` and `CCSN_CC_Form_Rot` no matter we use it or not.
- From your code, there is no limitation to use `CCSN_CC_Rot_Mag` and `CCSN_CC_Form_Rot` simultaneously. If you think there is, then you need to check the input runtime parameters.

Reply:
Yes. But there is `if (CCSN_CC_Rot)` condition above, it will be recorded in that case too.